pandas/tests/scalar/timedelta/test_constructors.py

```diff
@@ -40,8 +41,9 @@ def test_construction():
     assert Timedelta("1 hr") == timedelta(hours=1)
     assert Timedelta("1 hours") == timedelta(hours=1)
     assert Timedelta("-1 hours") == -timedelta(hours=1)
-    assert Timedelta("1 m") == timedelta(minutes=1)
-    assert Timedelta("1.5 m") == timedelta(seconds=90)
+    with tm.assert_produces_warning(FutureWarning, check_stacklevel=False):
+        assert Timedelta("1 m") == timedelta(minutes=1)
```

Comment:
there is nothing wrong here, I think you are catching m for some reason

Reply:
I thought we shouldnt change tests, hence the assert. Now I removed the assert and changed m to min.